File: snippets/fsharp/System/Version/Overview/example1.fs
module example1

// <Snippet6>
open System.Reflection

[<assembly: AssemblyVersionAttribute "2.0.1">]
do ()

type Example1 = class end

let thisAssem = typeof<Example1>.Assembly
let thisAssemName = thisAssem.GetName()
   
let ver = thisAssemName.Version
   
printfn $"This is version {ver} of {thisAssemName.Name}."
// The example displays the following output:
//        This is version 2.0.1.0 of Example1.
// </Snippet6>

File: snippets/csharp/System/Version/Overview/example1.cs
// <Snippet6>
using System;
using System.Reflection;

[assembly: AssemblyVersion("2.0.1")]

public class Example1
{
    public static void Main()
    {
        Assembly thisAssem = typeof(Example1).Assembly;
        AssemblyName thisAssemName = thisAssem.GetName();
        Version ver = thisAssemName.Version;

        Console.WriteLine($"This is version {ver} of {thisAssemName.Name}.");
    }
}

// The example displays the following output:
//        This is version 2.0.1.0 of Example1.
// </Snippet6>
